CHANGELOG.md

```diff
@@ -1,18 +1,25 @@
 # Changelog
 
-## Upcoming Breaking Changes
+## Project Deprecation
+Now that all primary Orion functionality has been merged into Tessera, Orion is being deprecated, 
```

Comment:
I feel this reads better if the sentence is split: "...Orion is being deprecated. We encourage all users with active..."